### DISCUSSION TITLE
Output Customiztion

### DISCUSSION THREAD

**Discussion by AgentHitmanFaris** [lllyasviel/Fooocus#2307] [Ideas]
![image](https://github.com/lllyasviel/Fooocus/assets/81744011/96dcb04c-10e6-42fc-b9d9-8a0e1e266e23)
Are this customization usefull for easy to use instead using
```
"path_outputs": "YourPath", at the config.txt

**Comment by mashb1t** [comment]
@AgentHitmanFaris not sure, i assume the majority of users doesn't switch output paths or paths in general often.
